R/HubbellSE.R

```diff
@@ -0,0 +1,122 @@
+#' Hubbell community simulation
+#'
+#' Neutral species abundances simulation turned into
+#' \linkS4class{SummarizedExperiment} according to the Hubbell model.
+#'
+#' @param N Integer scalar indicating the amount of different species initially
+#' in the local community
+#' @param M Integer scalar indicating the amount of different species in the
+#' metacommunity,including those of the local community
+#' @param I Integer scalar indicating the fixed amount of individuals in the
+#' local community (default: \code{I = 1000})
+#' @param d Integer scalar indicating fixed amount of deaths of local community
+#' individuals in each generation (default: \code{d = 10})
+#' @param m Numeric scalar indicating immigration rate: the probability that a
+#' death in the local community is replaced by a migrant of the metacommunity
+#' rather than by the birth of a local community member
+#' (default: \code{m = 0.02})
+#' @param tskip  Integer scalar indicating the number of generations that should
+#' not be included in the outputted species abundance matrix.
+#' (default: \code{tskip = 0})
+#' @param tend Integer scalar indicating the number of simulations
+#' to be simulated
+#' @param norm Logical scalar to indicate whether the time series should
+#' be returned with the abundances as proportions (norm = TRUE) or the raw
+#' counts (norm = FALSE, default)
+#'
+#' @importFrom SummarizedExperiment SummarizedExperiment
```

Comment:
Do we need this here if SummarizedExperiment is in DESCRIPTION file "Import:"

Reply:
So, should it be either in R file or in DESCRIPTION file? I checked mia package, and it looked like it is in both places.

Reply:
Well ok doesn't hurt to have it here.